scheduler/bin/run-docker.sh

```diff
@@ -33,6 +33,7 @@ docker run \
     --publish=${COOK_PORT}:${COOK_PORT} \
     -e "COOK_PORT=${COOK_PORT}" \
     -e "COOK_NREPL_PORT=${COOK_NREPL_PORT}" \
+    -e "COOK_FRAMEWORK_ID=cook-framework-${COOK_PORT}" \
```

Comment:
Quoting @wyegelwel from above:

> When a user upgrades and they don't configure the framework, Cook will connect to mesos with a different framework id and mark all jobs that were running failed because mesos won't have tasks for those jobs under the new framework id

So we want to use the same framework-id across restarts